### PR TITLE
Add 1.5.2 Release Notes

### PR DESCRIPTION
**Pull request by kuenishi**
After #988 merged.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  failure between Riak and Riak CS. Access logs are stored in
+  `console.log` with warning level.
+
+## Bugs Fixed
+
+- Fix PB connection pool (`pbc_pool_master`) leak
+  [riak_cs/#986](https://github.com/basho/riak_cs/pull/986) . Requests
+  asking for non-existent buckets without an authorization header
+  makes connections leak from the pool and eventually the pool goes
+  empty. This bug was introduced in relase 1.5.0.
```
Comment (shino) [inline RELEASE-NOTES.md]: One more API. Listing users.

**Comment by shino** [comment]
Repair script?

**Comment by kuenishi** [comment]
Fixed.

**Comment by shino** [comment]
From me, +1 by the last commit.

**Comment by kuenishi** [comment]
@borshop merge

**Pull request merged**